Source/NSFont.m

```diff
@@ -684,7 +684,7 @@ + (CGFloat) systemFontSizeForControlSize: (NSControlSize)controlSize
   
           if (fontSize == 0)
             {
-              fontSize = 6;
+              fontSize = 8;
```

Comment:
If we change this value here, we also need to adjust the documentation above (and maybe in other places as well).
But I don't think this is a good value, it is very close to the one use used for NSSmallFontSize (9). I need to test on a Mac what the returned sizes are there.

Reply:
I think 6 is just too small to render some readable text. I can drop this part cause I use 9 for NSMiniFontSize and 10 for NSSmallFontSize. I'll test these values on Mac too.

Reply:
On MacOS 10.15 NSControlSizeMini == 9, NSControlSizeSmall == 11. BTW, NSSmallControlSize and NSMiniControlSize is deprecated since 10.12.